genai-perf/genai_perf/subcommand/analyze.py

```diff
@@ -392,7 +392,9 @@ def _determine_infer_type(self):
         else:
             if self._args.concurrency:
                 infer_type = "concurrency"
-            else:
+            elif self._args.request_rate:
                 infer_type = "request_rate"
+            else:
+                infer_type = "concurrency"
```

Comment:
This could instead just check for request rate and set concurrency in its absence since it is the default mode.
Is there a reason that you explicitly check for both?

Reply:
No reason other than I feel like this is more explicit in showing intent of the code.

Reply:
I worry this introduces a spot where we can have a bug.

Reply:
How so? I feel like the code is safer/more future proof this way.

Reply:
There are two places that set concurrency. If we change it at one point, its two places to change.

Reply:
This is ultimately a method that is going to go away in the near future when we move to a different UX.